### ISSUE TITLE
Feature discussion: Automatic Image generation and deployment

### ISSUE THREAD

**Issue by tabedzki** [SpikeInterface/spikeinterface-dockerfiles#75]
Feature idea: a script that automatically generates and builds docker files for use with spikeinterface. 

Use case: 
- Automatic updating and versioning of spikesorters as they compile. For currently there are two images for kilosort4 when there have been 8 releases.

I think it would be useful to able to specify the exact version of a package someone might've used at a time and replicated it. I am open to thoughts on this idea.

**Comment by samuelgarcia** [comment]
Hi.
Thank you for the idea. This is very ambitious but could be doable for python based sorters.
We would need actions trigger by sorters releases.
The main case is indeed KS4.

And yes kilosort4 every week release was really hard to follow on our side!!!
